app/src/main/res/values-ru/placeholder.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please don't translate the `placeholder_` strings. They are just used to render a more useful preview of the screen layouts. Please remove this file from this commit.

Reply:
`placeholder` file removed